### PR TITLE
Add basic files to run CI

### PR DESCRIPTION
**Pull request by ruflin**

### CODE REVIEW

**Comment by ruflin** [comment]
@kuisathaverat Would be great to get this in the end hooked up to our CI. I copied the content from the package-registry and simplified it.

File: Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env groovy
+
+@Library('apm@current') _
+
+pipeline {
+  agent { label 'ubuntu && immutable' }
+  environment {
+    BASE_DIR="src/github.com/elastic/package-storage"
+    JOB_GIT_CREDENTIALS = "f6c7695a-671e-4f4f-a331-acdce44ff9ba"
```
Comment (ruflin) [inline Jenkinsfile:9]: @kuisathaverat Not sure if this is valid here?

Comment (kuisathaverat) [reply Jenkinsfile:9]: it is not used.

**Comment by kuisathaverat** [comment]
we need to add support to create jobs with JJBB too, I'll manage

**Comment by kuisathaverat** [comment]
Could you move the Jenkinsfile to the .ci folder to keep CI files inside .ci folder?

File: Jenkinsfile (hunk 1 above)
Comment (ruflin) [reply Jenkinsfile]: removed

**Comment by ruflin** [comment]
@kuisathaverat Just moved it and pushed a new commit. 

I guess this means we should also move it in the package-registry one day?

**Comment by kuisathaverat** [comment]
yep we have also to move it to JJBB, but with 7.7 there I would wait one week more 😄

**Comment by ruflin** [comment]
@kuisathaverat Anything else I need to do in this PR? Is there a way to test it directly in this PR as soon as the JJBB changes are merged?

**Comment by kuisathaverat** [comment]
no, there is nothing else to do here. After merging it, we can merge the JJBB PR in the infra repo and finally the https://github.com/elastic/package-storage/pull/6 to create the job in beats-ci

**Comment by ruflin** [comment]
@kuisathaverat Merged. Please ping me when all the others are in and I should test the setup.

**Pull request merged**